000-proposal-enhance-image-extension.md

```diff
@@ -0,0 +1,26 @@
+# Enhance Image Extension for Custom Authentication
```

Comment:
This should be properly modeled through a new authentication type in the CRD. That allows it to be maintainable in the long term. The `<COMPONENT>_ADDITIONAL_CONFIGURATION` does not offer proper support for a well designed API, its validation etc.